src/containers/UserLists/CreateUserListModal/container.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withFormik } from 'formik'
+import * as Yup from 'yup'
+import { connect } from 'react-redux'
+
+import { createListLoadingSelector } from 'Store/userMovieLists/selectors'
+import { createUserLits as createUserLitsAction } from 'Store/userMovieLists/actions'
+import CreateUserListModalComponent from './component'
+
+const regexp = /^[A-Za-z0-9\s]+$/
```

Comment:
extract to constants file

Reply:
@VladWithCookies this variable are used only for this component, should I move it to the constants file anyway?

Reply:
✅